src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -557,6 +570,19 @@ private File unpackWorkDir(Artifact artifact) throws MojoExecutionException {
         // If the artifact is a SNAPSHOT, then a.getVersion() will report the long timestamp,
         // but getFile() will be 1.1-SNAPSHOT.
         // Since getFile() doesn't use the timestamp, all timestamps wind up in the same place.
+	
+        // WRONG. getFile returns names like
+        // "lbfork-launch4j-3.53-20240105.004437-1-workdir-win32.jar" as of
+        // 2024-01-05.
+        // QUESTION: maybe it depends on Maven's version? Need to support both.
+        // FIX: if it contains expanded version replace it back by expandable version.
```

Comment:
What version of Java you have used to run the build? It can be the case

Reply:
Liberica 21, Maven 3.9.1